### ISSUE TITLE
[DOCS] 

### ISSUE THREAD

**Issue by foxfriends** [etsy/open-api#150]
**Link to Documentation Issue**
[OAuth Step 3: request an access token](https://developers.etsy.com/documentation/essentials/authentication/#step-3-request-an-access-token)

**What would you like to change?**
The example request body should be changed to show the actual request body in the expected format.

**Why are you proposing this change?**
The documentation says the request body should be in `application/x-www-form-urlencoded`, but then goes on to present an example request body that is... not in form-urlencoded format?

```
    {
      "grant_type"="authorization_code",
      "client_id"="1aa2bb33c44d55eeeeee6fff",
      "redirect_uri"="https://www.example.com/some/location",
      "code"="bftcubu-wownsvftz5kowdmxnqtsuoikwqkha7_4na3igu1uy-ztu1bsken68xnw4spzum8larqbry6zsxnea4or9etuicpra5zi",
      "code_verifier"="DSWlW2Abh-cf8CeLL8-g3hQ2WQyYdKyiu83u_s7nRhI"
    }
```

**Comment by etsyachristensen** [comment]
@foxfriends You are correct. That example is rather odd. We will work on updating it to make it more consistent.

**Comment by joey-lott** [comment]
Another discrepancy in this documentation example is that the code_verifier value shown is the challenge from the previous example. So surely it would make more sense and be consistent with rest of the documentation to use the code_verifier value from the previous example (i.e. the decoded value of the challenge) in this example.

**Comment by joey-lott** [comment]
to clarify, in this example the code_verifier value should probably be vvkdljkejllufrvbhgeiegrnvufrhvrffnkvcknjvfid
